### DISCUSSION TITLE
Would be lovely to have .md files on the github rendered in Obsidian

### DISCUSSION THREAD

**Discussion by mcgrue** [MrGVSV/obsidian-github-embeds#20] [Ideas]
(And it'd be even hotter if you could edit them in obsidian and a commit was made >_>)

**Comment by MrGVSV** [comment]
That's a great idea! I'll look into adding that!

As for editing the file, that's currently not something planned. It _might_ happen in the far future but it's not an immediate goal (and thus out of scope for this project unfortunately).
